src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const declarations = sourceString.trim().split(';');
```

Comment:
The chained methods 'trim' and 'split' should be on the same indentation level for better readability. [CODE STYLE]